frontend/rust-lib/flowy-core/src/deps_resolve/folder_deps.rs

```diff
@@ -300,13 +297,31 @@ impl FolderOperationHandler for DatabaseFolderOperation {
     })
   }
 
-  fn encoded_collab_v1(
-    &self,
-    _view_id: &str,
-    _layout: ViewLayout,
-  ) -> FutureResult<EncodedCollab, FlowyError> {
-    // Database view doesn't support collab
-    FutureResult::new(async move { Err(FlowyError::not_support()) })
+  fn get_encoded_collab_v1(&self, view_id: &str) -> FutureResult<EncodedCollabWrapper, FlowyError> {
+    let database_manager = self.0.clone();
+    let view_id = view_id.to_string();
+    FutureResult::new(async move {
+      let database_encoded_collab = database_manager
```

Comment:
You can directly read data from rocksdb. using database_manager will cause some unnecessary logic. For example, it will trigger sync for everying rows and database